kong/plugins/session/schema.lua

```diff
@@ -212,6 +212,20 @@ return {
               default = "session_logout"
             }
           },
+          {
+            hash_subject = {
+              description = "Whether to hash or not the subject when store_metadata is enabled.",
+              type = "boolean",
+              default = false
+            }
+          },
+          {
+            store_metadata = {
+              description = "Enables or disables to store metadata of sessions.",
```

Comment:
```suggestion
              description = "Enables or disables the storage of session metadata.",
```

Or maybe we can consider using the same description from [here](https://github.com/bungle/lua-resty-session/blob/525de6c5a0771a6638792681e6bb2648e9c122b6/README.md?plain=1#L322): 

```
Whether to also store metadata of sessions, such as collecting data of sessions for a specific audience belonging to a specific subject.         
```

Reply:
Of course.

Reply:
Thank you.

Reply:
By the way, could we also mention that `store_metadata` will only work when `storage == "kong"` to avoid future confusion?

Reply:
Good question. I mentioned scope limitations in the Engineering Brief. Please comment if you are free. Thank you.
https://docs.google.com/document/d/1r7nSZGOlTVhupoVGQ6Ymqsaw64eGHKSMibko8FgrwAQ/edit?tab=t.0#heading=h.lm7xbylqv511